submissions/lawalNurudeen/html-forms/contact-Us.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
```

Comment:
Kindly work it your already existing contact form page. Make the modification on that page

Reply:
@adaugochi it is done